pkgs/servers/misc/navidrome/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "navidrome";
+  version = "0.20.0";
+
+  src = fetchurl {
```

Comment:
```suggestion
  src = fetchzip {
```

Reply:
I can't do this becuse the archive does not contain a single directory/file, if I try I get `error: zip file must contain a single file or directory`